staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -522,28 +519,6 @@ func (c unstructuredCreator) New(kind schema.GroupVersionKind) (runtime.Object,
 	return ret, nil
 }
 
-type UnstructuredCopier struct{}
-
-func (UnstructuredCopier) Copy(obj runtime.Object) (runtime.Object, error) {
-	if _, ok := obj.(runtime.Unstructured); !ok {
-		// Callers should not use this UnstructuredCopier for things other than Unstructured.
-		// If they do, the copy they get back will become Unstructured, which can lead to
-		// difficult-to-debug errors downstream. To make such errors more obvious,
-		// we explicitly reject anything that isn't Unstructured.
-		return nil, fmt.Errorf("UnstructuredCopier can't copy type %T", obj)
-	}
-
-	// serialize and deserialize to ensure a clean copy
```

Comment:
@sttts this is the only one that concerns me.  DeepCopy of unstructured works the way we need it to now?

Reply:
I think we didn't have deepcopy funcs on Unstructured at the time of writing. Now we do, using structural recursion instead of encode+decode. So potentially the new deepcopy is also faster.